src/demo/App.js

```diff
@@ -1,17 +1,14 @@
 /* eslint no-magic-numbers: 0 */
 import React, {Component} from 'react';
-import PropTypes from 'prop-types';
-import * as R from 'ramda';
```

Comment:
Probably a good idea to remove these dependencies (or at least `ramda`) from the dependencies if they are never used.